CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### Bug Fixes
+
+* **copy-to-clipboard:** contrain max-width of component and min-width of copy icon ([202c3d9](https://github.com/harter-genesys/genesys-spark/commit/202c3d931fdad696968b72296ca28105b11f9858))
```

Comment:
I imagine this commit needs update, but it's a bit of a chicken an egg problem since it's coming from a fork.